packages/experiments/src/components/VirtualizedList/VirtualizedList.test.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import * as renderer from 'react-test-renderer';
-import { VirtualizedListBasicExample } from './examples/VirtualizedList.Basic.Example';
+import { VirtualizedList } from './VirtualizedList';
```

Comment:
Since the examples are moving to a separate package, this test within the experiments package can't render an example. So I duplicated the gist of the example below.